docker-compose.yaml

```diff
@@ -32,6 +34,8 @@ services:
       - perplexica-backend
     ports:
       - 3000:3000
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
This second `extra_hosts` is unnecessary.  Access to the LLM is only needed by the backend.